app/scripts/views/ready.js

```diff
@@ -57,6 +58,29 @@ function (Cocktail, BaseView, Template, Session, Xss, Strings,
       return this._createMarketingSnippet();
     },
 
+    /**
+     * Creates the service name for the ready template
+     * Can either be a string or a link to the relier redirectUri
+     *
+     * @returns {String} serviceName
+     * @private
+     */
+    _createServiceName: function () {
+      var relier = this.relier;
+      var serviceName = relier.get('serviceName');
+      var redirectUri = relier.get('redirectUri');
+      var autoRedirect = relier.get('autoRedirect');
+
+      // if this relier wants to always redirect the user then create a link to the service
+      if (autoRedirect && redirectUri && Url.isNavigable(redirectUri)) {
+        serviceName = Strings.interpolate('<a href="%s" class="no-underline" id="redirectTo">%s</a>', [
+          Xss.href(redirectUri), serviceName
+        ]);
```

Comment:
IIUC, clicking this link will not actually complete the oauth flow, since it doesn't contain the `code` or `state` parameters.  So we're relying on the relier to do something sensible when the user hits this page with no oauth state.

Given #2349 where Hello wants to use a different URL entirely for this link, I'm wondering whether the extra-query-parameter approach is really any simpler, or if we should just go ahead and add a `service_uri` or similar field to the relier metadata.

I guess we can always iterate towards that in a future train if it turns out to make sense.